user_guide_src/source/installation/upgrade_responses.rst

```diff
@@ -0,0 +1,47 @@
+Upgrade HTTP Responses
+######################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+==============
+- `Output Class Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/libraries/output.html>`_
+- :doc:`HTTP Responses Documentation Codeigniter 4.X </outgoing/response>`
+
+What has been changed
+=====================
+- The methods have been renamed
+
+Upgrade Guide
+=============
+1. The methods in the HTML Responses class are named slightly different. The most important change in the naming is the switch from underscored method names to camelCase. The method ``set_content_type()`` from version 3 is now named ``setContentType()`` and so on.
```

Comment:
"the HTML Responses class" -> "the Response class" or "the HTTP Response class" ?

Reply:
Should be HTTP